config/xsl/diffreport/util/timer-table.xsl

```diff
@@ -0,0 +1,116 @@
+                        <th class="table-sortable:numeric" title="The standard deviation of all data within this data series.">Dev.</th>
+                        <xsl:for-each select="$summaryElement/percentiles/*">
+	                        <th class="table-sortable:numeric colgroup1" title="The nth percentile of the data series.">
+	                            <xsl:value-of select="translate(name(), 'p', 'P')"/>
+	                        </th>
+	                    </xsl:for-each>
+                    </tr>
+                </thead>
+                <xsl:variable name="count" select="count($elements)" />
+                <xsl:choose>
+                    <xsl:when test="$count > 0">
+                        <tfoot>
+                            <xsl:for-each select="$summaryElement">
+                                <xsl:call-template name="timer-summary-row">
+                                    <xsl:with-param name="type" select="$type"/>
+                                    <xsl:with-param name="rows-in-table" select="$count"/>
+                                </xsl:call-template>
+                            </xsl:for-each>
+                            <xsl:call-template name="filtered-footer-row" />
+                        </tfoot>
+                        <tbody>
+                            <xsl:for-each select="$elements">
+                                <xsl:sort select="name" data-type="number"/>
+                                <xsl:call-template name="timer-row">
+                                    <xsl:with-param name="type" select="$type"/>
+                                </xsl:call-template>
+                            </xsl:for-each>
+                        </tbody>
+                    </xsl:when>
+                    <xsl:otherwise>
+                        <xsl:variable name="columns">
+	                        <xsl:choose>
+	                            <xsl:when test="$type = 'request'">
+	                                <xsl:value-of select="11 + $percentileCount"/>
+	                            </xsl:when>
+	                            <xsl:when test="$type = 'transaction'">
+	                                <xsl:value-of select="12 + $percentileCount"/>
+	                            </xsl:when>
+	                            <xsl:when test="$type = 'action'">
+	                                <xsl:value-of select="12 + $percentileCount"/>
+	                            </xsl:when>
+	                            <xsl:otherwise>
+	                                <xsl:value-of select="11 + $percentileCount"/>
+	                            </xsl:otherwise>
+	                        </xsl:choose>
+                        </xsl:variable>
+                        <tfoot>
+                            <tr>
+                                <td colspan="{$columns}"></td>
+                            </tr>
+                            <xsl:call-template name="filtered-footer-row" />
+                        </tfoot>
```

Comment:
When using a single td with colspan, the column (set) striping effect as introduced in the table header is lost. We should have at least one td per column (set). Make sure to match the look with the Bandwidth table.

(I know that column striping in general is a mess in the diff report, but definitely another topic. Just wanted to make sure that we don't add to it.)